code/__DEFINES/logging.dm

```diff
@@ -51,6 +51,7 @@
 #define LOG_CLONING (1 << 22)
 #define LOG_MECHCOMP (1 << 23)
 #define LOG_BLACKMARKET (1 << 24)
+#define LOG_NTSL (1 << 20)
```

Comment:
we already have a 1<<20 needs to be 1<<25

Reply:
I am pretty sure I changed this; will fix.